web/beacon-app/src/features/members/components/MemberDetails.tsx

```diff
@@ -51,27 +51,25 @@ export default function MemberDetails() {
           fallback={<div>We are unable to fetch your member, please try again.</div>}
         >
           <div className="my-10">
-            <div className="flex items-center justify-between rounded-md bg-[#F7F9FB] px-6 py-3">
-              <Heading as="h1" className="flex w-40 items-center gap-5 text-2xl font-semibold">
-                <span className="mr-2">
+            <div className="flex items-center justify-between rounded-md bg-[#F7F9FB] px-6 py-4">
+              <Heading as="h1" className="text-[16px] font-semibold leading-[25px]">
```

Comment:
Although the Tailwind class `text-base` states the font-size should be `16px`, I saw `14px` in my browser. Specifying the font-size here ensures that it's always at `16px`.